src/components/Dashboard/Dashboard.jsx

```diff
@@ -85,10 +90,11 @@ export default function Dashboard() {
             </div>
             {mobileView && <FiltersFlat />}
             <div className="posts">
-              <Post />
-              <Post />
-              <Post />
-              <Post />
+              {posts ? (
+                posts.map((post) => <Post post={post} />)
+              ) : (
+                <CircularProgress />
+              )}
```

Comment:
Isn't this duplicated at the end of the file? Also, why is it here?

Reply:
Fixed 783ee049a3396f564058a194d966ebf9d3ca9f7e